ext/bin/require-jdk

```diff
@@ -77,24 +77,28 @@ esac
 curl -sSLo "$tmpdir/jdk.tar.gz" \
      "https://api.adoptopenjdk.net/v2/binary/releases/$ver?os=$adoptos&arch=x64&openjdk_impl=hotspot&release=latest&type=jdk"
 
-rm -rf "$install/jdk"
-mkdir -p "$install"
+(cd "$tmpdir"
+ mkdir tmp-unpack
+ cd tmp-unpack
+ tar xpSf ../jdk.tar.gz)
 
 case "$OSTYPE" in
     darwin*)
-        (cd "$install"
-         tar xpSf "$tmpdir/jdk.tar.gz"
+        (cd "$tmpdir/tmp-unpack"
          case "$ver" in
-             openjdk8) mv */Contents/Home jdk ;;
-             *) mv * jdk ;;
+             openjdk10) mv * jdk ;;
+             *) mv */Contents/Home jdk ;;  # At least 8 and 11
          esac)
         ;;
     *)
-        (cd "$install"
+        (cd "$tmpdir/tmp-unpack"
```

Comment:
Is this a duplicate of line 80-83?

Reply:
I don't think so?  I'm trying not to change the current dir globally, so I cd in a subshell each time. i.e. the case statement here just handles the platform archive specific naming issues.